test/bls.test.ts

```diff
@@ -1,22 +1,32 @@
-import { TestBlsInstance } from "../types/truffle-contracts";
+import { TestBlsFactory } from "../types/ethers-contracts/TestBlsFactory";
+import { TestBls } from "../types/ethers-contracts/TestBls";
+import web3 from "web3";
+import { assert } from "chai";
 
-const TestBLS = artifacts.require("TestBLS");
 import * as mcl from "./utils/mcl";
-import { bn, bnToHex, ZERO } from "./utils/mcl";
+import { ethers } from "@nomiclabs/buidler";
 
 describe("BLS", async () => {
-    let bls: TestBlsInstance;
+    let bls: TestBls;
     before(async function() {
         await mcl.init();
-        bls = await TestBLS.new();
+        const accounts = await ethers.getSigners();
+        bls = await new TestBlsFactory(accounts[0]).deploy();
+        await bls.deployed();
```

Comment:
This is an example of using buidler/typechain ether-v4. Note that:
- We get signers from the buidler environment. We `import { ethers } from "@nomiclabs/buidler"`
- We use the factory generated by typechain/ethers-v4 to deploy contract.
- Remember to `await contact.deployed();`